src/components/DataTable/index.js

```diff
@@ -52,10 +63,20 @@ export function DataTable ({
 		}, 500);
 	}, [filterValue]);
 
+	// rounding page number divisible by 5
+	function round(x) {
```

Comment:
@Aringoaway Pls rename this function to something more understandable. When you think of it, what does this function do? It certainly does not "round", but rather returns count of the rows in datatable